script.js

```diff
@@ -1,4 +1,199 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+import { DECK } from "./deck.js";
+let cardDeck = DECK;
+const GAMESTATES = ["draw-cards", "stand-or-hit", "show-results"];
+let gameState = GAMESTATES[0];
```

Comment:
It would be better if you game each game state a variable instead. Reason being, ```GAMESTATES[0]``` does not have much meaning and it'll be harder to debug/understand